src/App.tsx

```diff
@@ -24,6 +24,11 @@ import MedlHistorikk from "./pages/medlemskap-i-folketrygden/MedlHistorikk";
 import { WithAuth } from "./store/providers/WithAuth";
 
 export const basePath = "/person/personopplysninger";
+
+// Import this early, to ensure our own CSS gets higher specificity
+import "@navikt/ds-css/dist/index.css";
+import moment from "moment";
```

Comment:
Om funksjonaliteten vi bruker fra Moment finnes i dayjs, date-fns eller Luxon, tror jeg vi kan slanke koden en del. https://npmtrends.com/date-fns-vs-dayjs-vs-luxon-vs-moment

Reply:
Foreslår at vi lager det som en egen oppgave i såfall. Så slipper vi scope creep på denne. Men godt poeng! Det er mye å dra inn motion for bare litt datoformattering.